docs/dummies.md

```diff
@@ -76,10 +76,14 @@ of approaches in turn, until one succeeds:
   `Lazy<TValue>` whose `Value` is a Dummy of type
   `TValue`, or a default `TValue` if no Dummy can be made
   for `TValue`.
-1. If `T` is [fakeable](what-can-be-faked.md), the Dummy will be a
-  Fake `T`.
+1. If `T` is a tuple type (`Tuple<>` or `ValueTuple<>`), the Dummy will
+be a tuple whose elements are dummies, or default values when dummies
+can't be made.
+  `Activator.CreateInstance`.
 1. If `T` is a value type, the Dummy will be a `T` created via
   `Activator.CreateInstance`.

```

Comment:
These two were out of order! We try the value type strategy before the fake strategy.

Reply:
Probably my fault. I reordered some of the strategies during the performance work, if I recall correctly. Thanks for correcting!